src/SkyCoords.jl

```diff
@@ -115,6 +134,12 @@ rotmat(::Type{<:ICRSCoords}, ::Type{<:GalCoords}) = GAL_TO_ICRS
 rotmat(::Type{<:ICRSCoords}, ::Type{<:ICRSCoords}) = I
 rotmat(::Type{<:GalCoords}, ::Type{<:GalCoords}) = I
 rotmat(::Type{<:FK5Coords{e}}, ::Type{<:FK5Coords{e}}) where {e} = I
+rotmat(::Type{<:GalCoords}, ::Type{<:SuperGalCoords}) = GAL_TO_SUPERGAL
+rotmat(::Type{<:SuperGalCoords}, ::Type{<:GalCoords}) = SUPERGAL_TO_GAL
+rotmat(::Type{<:SuperGalCoords}, ::Type{<:SuperGalCoords}) = I
+rotmat(::Type{<:SuperGalCoords}, ::Type{<:ICRSCoords}) = SuperGal_TO_ICRS
+rotmat(::Type{<:ICRSCoords}, ::Type{<:SuperGalCoords}) = ICRS_TO_SuperGAL
```

Comment:
The other part is about `rotmat`. I think the mistake there was that `rotmat` accepts arguments in `(to, from)` format, but the opposite matrices were given. So, for lines 137–141 in src/SkyCoords.jl, please change it with the following snippet:
```suggestion
rotmat(::Type{<:SuperGalCoords}, ::Type{<:SuperGalCoords}) = I
rotmat(::Type{<:GalCoords}, ::Type{<:SuperGalCoords}) = SUPERGAL_TO_GAL
rotmat(::Type{<:SuperGalCoords}, ::Type{<:GalCoords}) = GAL_TO_SUPERGAL
rotmat(::Type{<:SuperGalCoords}, ::Type{<:ICRSCoords}) = ICRS_TO_SUPERGAL
rotmat(::Type{<:ICRSCoords}, ::Type{<:SuperGalCoords}) = SUPERGAL_TO_ICRS
```